partiql-parser/src/scanner.rs

```diff
@@ -349,6 +385,174 @@ mod test {
             syntax_error("IGNORED MESSAGE", Position::at(1, 32)),
         ]
     )]
+    #[case::numeric_literals(
+        "1 -0099 1.1 +00055.023100 99.1234e0010",
+        vec![
+            Ok(Token {
+                content: Content::IntegerLiteral(1.into()),
+                start: LineAndColumn::at(1, 1),
+                end: LineAndColumn::at(1, 2),
+                text: "1",
+                remainder: Remainder {
+                    input: " -0099 1.1 +00055.023100 99.1234e0010",
+                    offset: LineAndColumn::at(1, 2)
+                }
+            }),
```

Comment:
I think it is getting time to add a macro to make these cases easier to write...

Reply:
Tracking as #21.